TaskarCenterAtUW/TDEI-infra-as-code
# Enabled diagnostic settings

**Changes made:**
1. Created log analytics workspace - stored the workspace id in keyvault
2. Configured diagnostic settings for app services to use the workspace id created in step 1
3. Added code to provision diagnostic settings for app services
4. Added info in README files for the changes made
5. README for keyvault updated with list of keys provisioned
6. Added code to set loggerdb connection string in the keyvault

**How to Test**
1. Delete all resources except for the `keyVault` "tdei-rkd1907" in the resource Group: GaussianRG-rkd1907
2. Check out the code (or) alternatively, one can run github workflow to provision infrastructure
3. If running on local machine, switch to this branch, and follow the instructions in README file. `py create-env.py -e rkd1907 -c dev -l eastus`
5. If running on github ui, choose this branch to execute the workflow from, give "rkd1907" as the environment to create
6. After the script  / workflow executes, everything should be in place
